label_studio/tasks/migrations/0029_task_alter_foreign_key_actions.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from django.db import migrations
+from django.contrib.postgres.operations import TrigramExtension
+from django.db import connection
+
+logger = logging.getLogger(__name__)
+
+
+def forwards(apps, schema_editor):
```

Comment:
it's very unclear what is going on here, please make a good comment

Reply:
Added